pre-docs/bolt_configure_puppetdb.md

```diff
@@ -0,0 +1,44 @@
+
+# Configuring PuppetDB
+
+Configure bolt to connect to PuppetDB
+
+## Why
+
+Bolt can use PuppetDB to (generate an inventory
+file)[inventory_file_generating.md], (gather target
+facts)[writing_plans.md#pdb_facts], or (Collecting
+Data)[writing_plans.md#pdb_query]
+
+## Configuration
+
+To configure the bolt puppetdb client add a `puppetdb` section to your
+inventory including the `server-urls` and path to the puppetdb `cacert`. To
```

Comment:
What is server-urls and how do I know what to set it to?  Unless we're assuming intermediate puppet users, in which case might not need to include that.